state_machine/states/impl/waypoint_impl.py

```diff
@@ -123,7 +125,10 @@ async def run(self: Waypoint) -> State:
                 # Gradually move toward goal altitude
                 curr_altitude += altitude_slope * line_segment.length()
 
-                await move_to(self.drone.system, lat_deg, lon_deg, curr_altitude, 1.0)
+                try:
+                    await move_to(self.drone.system, lat_deg, lon_deg, curr_altitude, 1.0)
+                except ActionError:
+                    logging.warning(ActionResult.Result.TIMEOUT)
```

Comment:
I sent the logs in the flight channel in the Discord, verify that this is the correct built-in